common/launch_test.go

```diff
@@ -3,46 +3,42 @@ package common
 import (
 	"testing"
 
-	"github.com/grafana/xk6-browser/k6ext/k6test"
-
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/xk6-browser/k6ext/k6test"
 )
 
 func TestLaunchOptionsParse(t *testing.T) {
 	t.Parallel()
 
-	testCases := []struct {
-		name   string
+	for name, tt := range map[string]struct {
 		opts   map[string]any
-		assert func(*testing.T, *LaunchOptions)
+		assert func(testing.TB, *LaunchOptions)
 	}{
 		// TODO: Check other options.
-		{
-			name: "args",
+		"args": {
 			opts: map[string]any{
 				"args": []any{"browser-arg1='value1", "browser-arg2=value2", "browser-flag"},
 			},
-			assert: func(t *testing.T, lopts *LaunchOptions) {
-				require.Len(t, lopts.Args, 3)
-				assert.Equal(t, "browser-arg1='value1", lopts.Args[0])
-				assert.Equal(t, "browser-arg2=value2", lopts.Args[1])
-				assert.Equal(t, "browser-flag", lopts.Args[2])
+			assert: func(tb testing.TB, lo *LaunchOptions) {
+				tb.Helper()
+				require.Len(tb, lo.Args, 3)
+				assert.Equal(tb, "browser-arg1='value1", lo.Args[0])
+				assert.Equal(tb, "browser-arg2=value2", lo.Args[1])
+				assert.Equal(tb, "browser-flag", lo.Args[2])
 			},
 		},
-	}
-
-	for _, tc := range testCases {
-		tc := tc
-		t.Run(tc.name, func(t *testing.T) {
+	} {
+		tt := tt
+		t.Run(name, func(t *testing.T) {
 			t.Parallel()
-
-			vu := k6test.NewVU(t)
-			opts := vu.ToGojaValue(tc.opts)
-			lopts := NewLaunchOptions()
-			err := lopts.Parse(vu.Context(), opts)
-			require.NoError(t, err)
-			tc.assert(t, lopts)
+			var (
+				vu = k6test.NewVU(t)
+				lo = NewLaunchOptions()
+			)
+			require.NoError(t, lo.Parse(vu.Context(), vu.ToGojaValue(tt.opts)))
```

Comment:
I actually prefer the way the test was laid out before. I think having too much on one line puts more cognitive load on the reader. Happy to leave this as is though.

Reply:
Yeah, sure.

Reply:
I'm resolving this since your other suggestion is similar.